README.md

```diff
@@ -49,7 +49,18 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
-You can find more thorough guides in the [docs folder](https://github.com/theforeman/forklift/tree/master/docs).
+In case using vagrant is not desired, ansible playbooks and roles from this repo can be used separately. This is useful if existing host should be used for the installation, e.g. a beaker machine. In order to deploy devel environment on host test.example.com, following should be executed:
```

Comment:
I think one needs to also manually add `vagrant` user to sudoers file, otherwise the actions requiring privileged access will fail.
- i workarounded these problems by running `useradd vagrant` and adding it to sudoers file using `NOPASSWD` directive before runnign the playbook.

Reply:
got to the same conclusion, for some reason https://github.com/theforeman/puppet-katello_devel/blob/master/templates/sudoers-dropin.erb didn't work, I'll add it to this docs for now but we should fix it in ansible playbook/role later